BLSQ/iaso
# IA-3206-IA-3207: Add and move projects columns and filter on form and submission page

Explain what problem this PR is resolving
- Add, remove and move columns and add projects filter on form page
- Add, remove  columns and add projects filter on submissions page
Related JIRA tickets : [IA-3206](https://bluesquare.atlassian.net/browse/IA-3206) [IA-3207](https://bluesquare.atlassian.net/browse/IA-3207)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add, remove and move columns and add projects filter on form page and also change the XLS and XML download button
- Add, remove  columns and add projects filter on submissions page


## How to test
- Go to form page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx) and check the XLS & XML download button
- Go to submission page and check if it match the [mockup](https://whimsical.com/multi-project-web-interfaces-modifications-LRbazNKxybyqpULUQiKcJx)


## Print screen / video
![Screenshot-from-2024-08-08-17-49-26](https://github.com/user-attachments/assets/e34f5cad-95e7-427d-800a-ac92d2a9dfe3)
![Screenshot-from-2024-08-08-17-49-40](https://github.com/user-attachments/assets/d785f1bb-a92a-46c9-b43d-d3444922936a)

Upload here print screens or videos showing the changes

## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3206]: https://bluesquare.atlassian.net/browse/IA-3206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[IA-3207]: https://bluesquare.atlassian.net/browse/IA-3207?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
